app/models/event.rb

```diff
@@ -21,6 +21,8 @@ class Event < ApplicationRecord
 
   scope :active, -> { where('deadline >= ?', Date.current) }
 
+  has_and_belongs_to_many :users
```

Comment:
pretty sure you can name an association with 
has_and_belongs_to_many :participants, source => :users

Reply:
What else does it change in the Code?

Reply:
Anywhere where you're using event.users to get the participants would change to event.participants